core/src/test/scala/io/qbeast/core/model/QuerySpaceFromToTest.scala

```diff
@@ -1,100 +1,116 @@
 package io.qbeast.core.model
 
-import io.qbeast.core.transform.LinearTransformation
+import io.qbeast.core.transform.{LinearTransformation, Transformation}
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
 
 class QuerySpaceFromToTest extends AnyFlatSpec with Matchers {
 
-  "QuerySpace from to" should "intersect correctly" in {
+  val toSpaceCoordinates
```

Comment:
Better to declare this variable as private, since you are not using it outside the class.

Reply:
I should change the variable names to columnsFrom and columnsTo, since the usage of _ as a prefix is a Java convention of bean getters/setters, and its usage as suffixes is discouraged:

Quoting from [Scala style guide](https://docs.scala-lang.org/style/naming-conventions.html):
> The underscore is in the prefix position rather than the suffix to avoid any danger of mistakenly typing name _ instead of name_.